src/fragment.ts

```diff
@@ -0,0 +1,21 @@
+import { TramOneComponent } from './types';
+import { registerDom } from './dom';
+import useEffect from './use-effect';
+
+const xml = registerDom(null);
+
+const fragment: TramOneComponent = (props, children) => {
+	useEffect((ref) => {
+		(children || []).forEach((child) => {
+			if (typeof child === 'string') {
+				ref.insertAdjacentText('beforebegin', child);
+			} else {
+				ref.insertAdjacentElement('beforebegin', child);
+			}
+		});
+		ref.remove();
+	});
+	return xml`<tram-fragment>${children}</tram-fragment>`;
```

Comment:
`<tram-fragment>` isn't a real tag per-se, rather it's just a placeholder that will be immediately removed at the end of the `useEffect`.